website/docs/d/contract_subject_one_way_filter.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+subcategory: "Contract"
+layout: "aci"
+page_title: "ACI: aci_contract_subject_one_way_filter"
+sidebar_current: "docs-aci-contract-subject-one-way-filter"
+description: |-
+  Data source for ACI Filter
+---
+
+# aci_filter #
+
+Data source for ACI Filter.
+
+NOTE: The aci_contract_subject_one_way_filter should only be used with datasources as it will create conflicts with the aci_contract_subject resource
```

Comment:
This is a datasource so this message is not needed here. Just in the resource doc.